docs/developer/checkout.mdx

```diff
@@ -274,23 +275,30 @@ As a result, we get the payment object:
 
 ## Completing the checkout
 
-This operation requires only the checkout ID as an input. Its purpose is to ensure this checkout is correct. To do this, it verifies if:
+The purpose of the operation is to ensure this checkout is correct. To do this, it verifies if:
 
 1. The required addresses are valid.
 
 2. All selected products are in stock (while making the purchase, another user could already buy the last available item).
 
 3. The payment creation succeeded.
 
+4. No additional actions are required.
+
 If these are satisfied, the checkout is transformed into an order and the customer receives a confirmation email.
 
 The `checkoutComplete` mutation requires the following input:
 
 - `checkoutId`: the ID of the checkout to complete.
+- `storeSource`: determines whether to store the payment source for future usage.
```

Comment:
I think it is only supported by Stripe and it would for mentioning here.

Reply:
AFAIK It is supported by Paymentinterface as we pass it to the payment gateway. In that case, we should include it and explain.